_data/people/beomki-yeo.yml

```diff
@@ -0,0 +1,9 @@
+name: Beomki Yeo
+shortname: beomki-yeo
+title: Post-doctoral researcher
+active: true
+institution: University of California, Berkeley
+website:
+photo: /assets/images/team/beomki-yeo.png
+e-mail: beomki.yeo@berkeley.edu
+presentations:
```

Comment:
```suggestion
```

@beomki-yeo :wave: Welcome to the team! Can you please rebase this PR off of `master` to pick up Carlo's PR and avoid the merge conflict and then also remove this presentations field (I think that it being empty is causing the CI to fail).

Reply:
Done! but CI is still failing :(((